docs/Maestro/monitoring.md

```diff
@@ -75,7 +95,7 @@ $ maestro status ./tests/lulesh --layout narrow
 A snippet of the narrow layout for the above study is shown below.  These layouts are computed by status command, so you can alternate between them in the same study without issue:
 
 <figure markdown>
-  ![Flat Layout Lulesh Cancelled](../assets/images/examples/study_monitoring/narrow_layout_lulesh_cancelled.png)
+  ![Narrow Layout Lulesh Cancelled](../assets/images/examples/study_monitoring/narrow_layout_lulesh_cancelled.png)
```

Comment:
I think this image might be missing? -- at least it doesn't show up in the list of changed files on my end.

Reply:
I didn't add the image here, I just modified the label since I noticed it was incorrect